.github/workflows/ci.yml

```diff
@@ -166,12 +166,6 @@ jobs:
           env TEST_LANG="cpp" NIMFLAGS="${NIMFLAGS} --gc:refc" nimble test
           if [[ "${{ matrix.branch }}" == "devel" ]]; then
             echo -e "\nTesting with '--gc:orc':\n"
-            if env TEST_LANG="c" NIMFLAGS="${NIMFLAGS} --gc:orc" nimble test; then
-              echo "Nim devel with --gc:orc works again! Please remove this check in ci.yml"
-              false
-            fi
-            if env TEST_LANG="cpp" NIMFLAGS="${NIMFLAGS} --gc:orc" nimble test; then
-              echo "Nim devel with --gc:orc works again! Please remove this check in ci.yml"
-              false
-            fi
+            env TEST_LANG="c" NIMFLAGS="${NIMFLAGS} --gc:orc" nimble test
```

Comment:
orc is default in devel, it would be more relevant to test devel-refc separately

Reply:
oh, we do that above - well, I'd remove --gc:orc regardless to catch "defaults" whatever they might be

Reply:
The .nimble file also handles this: https://github.com/status-im/nim-ssz-serialization/blob/master/ssz_serialization.nimble

It should go in one place or another, but don't have strong view on where.

But I'm not sure what you mean by "test devel-refc separately": the idea is that as long as Status hasn't transitioned away from refc, anyversion-refc will be relevant. So both ORC and refc should be tested, post-1.6. Neither is any more "separate" than the other.

Reply:
Sure, one can also have implied-default-model and refc separately, post-1.6.

If Status also starts depending on ORC, then that then needs to be added specifically, regardless of defaults, but it doesn't yet.

Reply:
Oh, it doesn't actually work in ORC in `devel` (but apparently does in 2.0):
```
[Suite] Merkle proofs
  [OK] get_branch_indices
  [OK] get_path_indices
  [OK] get_helper_indices
  [OK] verify_merkle_multiproof
Traceback (most recent call last)
/home/runner/.nimble/pkgs/unittest2-0.0.6/unittest2.nim(757) test_ssz_optional
/home/runner/.nimble/pkgs/unittest2-0.0.6/unittest2.nim(682) suiteEnded
/home/runner/.nimble/pkgs/unittest2-0.0.6/unittest2.nim(309) suiteEnded
/home/runner/work/nim-ssz-serialization/nim-ssz-serialization/nim/lib/system/arc.nim(211) isObjDisplayCheck
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
  [OK] is_valid_merkle_branch
```
So not sure why the initial runs triggered the "re-enable ORC CI" message, which is/was supposed to run anyway only when ORC actually runs.

Fixing whatever's happening there is orthogonal to the gist of this PR.

Reply:
The explicit `--gc:orc` is needed for the 1.6 branch, it is only default from 2.0 onwards, no?

Reply:
Ah, `if [[ "${{ matrix.branch }}" == "devel" ]]; then`, got it

Reply:
About "The .nimble file also handles this", I think Nim regularly builds our projects using the nimble file, so keeping it erroring there makes them aware of the issue if they try to make our projects build, while putting it in our custom CI allows us to still have a green CI with known errors suppressed for ourselves.

Reply:
That's the status quo and also the situation this PR would perpetuate that
https://github.com/status-im/nim-ssz-serialization/blob/3f8946ab2de304a9130aa80874bc2e4025d62303/ssz_serialization.nimble#L28-L33

This PR doesn't change that, and it accomplishes what you suggest, and which I agree with.